profiles/galaxy_base/init.sh

```diff
@@ -54,11 +54,33 @@ setup_signing_service() {
     fi
 }
 
+setup_container_signing_service() {
+
+    if ! skopeo --version > /dev/null; then
+        log_message 'WARNING: skopeo is not installed. Skipping container signing service setup.'
+        return
+    fi
+
+    log_message "Setting up container signing service."
+    export KEY_FINGERPRINT=$(gpg --show-keys --with-colons --with-fingerprint /src/galaxy_ng/dev/common/ansible-sign.key | awk -F: '$1 == "fpr" {print $10;}' | head -n1)
+    export KEY_ID=${KEY_FINGERPRINT: -16}
+    gpg --batch --import /src/galaxy_ng/dev/common/ansible-sign.key &>/dev/null
```

Comment:
please don't hide standard out. I find that this makes debugging really had to do when things break